capitals.rb

```diff
@@ -150,3 +153,27 @@
     name: "Wyoming",
     capital: "Cheyenne"
 }]
+
+num_right = 0
+num_wrong = 0
+states = states.shuffle
+
+while true
+      states.length.times do |state|
+        puts "What's the capital of " + states[state][:name] + "?"
+        response = gets.chomp
+        if response == states[state][:capital]
+          num_right += 1
+          puts "Correct!"
+          puts "SCORE::: Correct: " + num_right.to_s + " Wrong: " + num_wrong.to_s
+        else
```

Comment:
Not a bad way to convert to a string, but the more Rubyist way is to use string interpolation.